exp/api/v1beta1/azuremanagedmachinepool_types.go

```diff
@@ -44,6 +44,11 @@ type NodePoolMode string
 // AzureManagedMachinePoolSpec defines the desired state of AzureManagedMachinePool.
 type AzureManagedMachinePoolSpec struct {
 
+	// AdditionalTags AdditionalTags is an optional set of tags to add to Azure resources managed by the
```

Comment:
Is it correct to update just the `v1beta1` file ?
was I supposed to backport this to the `v1alpha*` versions ?
or was I supposed to create a `v1beta2` version when introducing this change ?

Reply:
As the APIs are still experimental, I think it should be fine to add in v1beta1 alone.
cc @mtougeron @jackfrancis to comment more on this one.

Reply:
You'll want to change `v1beta1` types + conversion to and from alpha API versions

As a general rule, we only update the latest API version types on the main branch. The alpha types are there for backwards compatibility purposes (conversion to and from the old types).